AGGREGATE.md

```diff
@@ -303,7 +303,8 @@ shared.
 
 The encryption will use public keys specified by the aggregation service. The
 browser will encrypt payloads just before the report is sent by fetching the
-public key endpoint with an un-credentialed request. The processing origin will
+public key endpoint (the aggregation service coordinator origin at the path
+ `.well-known/aggregation-service/publick-keys`) with an un-credentialed request. The processing origin will
```

Comment:
Should be 

`.well-known/aggregation-service/public-keys`

Reply:
This was already fixed in 6e676b61dc067f5fb472afbec847e8706d553d7a.